Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -125,11 +104,20 @@ open class MediaAttachment: NSTextAttachment {
         overlayImage = nil
     }
 
+    /// Image to be displayed: Contains the actual Asset + the overlays (if any), embedded
+    ///
     internal var glyphImage: UIImage?
 
+    /// Attachment's Delegate
+    ///
     weak var delegate: MediaAttachmentDelegate?
-    
-    var isFetchingImage: Bool = false
+
+    /// Indicates if there's a download OP in progress, or not.
+    ///
+    fileprivate var isFetchingImage = false
```

Comment:
Confirmed to be safe! WPiOS isn't hooked up to that flag (which should always have been internal!)